docs/options.md

```diff
@@ -494,19 +494,23 @@ Platform-specific environment variables are also available:<br/>
 
     ```yaml
     # Set some compiler flags
-    CIBW_ENVIRONMENT: "CFLAGS='-g -Wall' CXXFLAGS='-Wall'"
+    CIBW_ENVIRONMENT: CFLAGS='-g -Wall' CXXFLAGS='-Wall'
 
     # Append a directory to the PATH variable (this is expanded in the build environment)
-    CIBW_ENVIRONMENT: "PATH=$PATH:/usr/local/bin"
+    CIBW_ENVIRONMENT: PATH=$PATH:/usr/local/bin
+
+    # Prepend a directory containing spaces on Windows.
+    CIBW_ENVIRONMENT_WINDOWS: >-
```

Comment:
This should be added to the config tab too, and I'm pretty sure there's no need to strip the surrounding white space, so I'd drop the "-" for consistency with the other examples.

Reply:
I'm no toml expert: is there a way to emit a `\\` without using a `\\\\`?
```
In [34]: print(toml.loads(r"""
    ...: [tool.cibuildwheel.windows]
    ...: environment = { PATH="C:\\\\Program Files\\\\PostgreSQL\\\\13\\\\bin;$PATH" }
    ...: """)["tool"]["cibuildwheel"]["windows"]["environment"]["PATH"])
C:\\Program Files\\PostgreSQL\\13\\bin;$PATH
```
(I would rather use a `[tool.cibuildwheel.windows.environment]` here but I guess it messes up the "narrative")

Reply:
(`>-` -> `>` fixed)

Reply:
Use single quotes, that should cut that down.

```
>>> print(toml.loads(r"""
...   [tool.cibuildwheel.windows]
...   environment = { PATH='C:\\Program Files\\PostgreSQL\\13\\bin;$PATH' }
... """)["tool"]["cibuildwheel"]["windows"]["environment"]["PATH"])
```

Reply:
I tried that but probably I was doing something wrong. Added, thank you!